scripts/run_module.py

```diff
@@ -105,21 +116,26 @@ def run_module(
         )
 
     module_name = module_config_path.parent.stem
-    volumes = [f'{(repo_root_dir / "samples").resolve()}:/opt/savant/samples']
+    if module_name == 'module':
+        module_name = module_config_path.parent.parent.stem
```

Comment:
Is this to account for dir structure as in `yolov8seg` sample?

`face_reid` is left unsupported currently.

Let's make it more generalized: go up in dir tree until `samples` is found, then previous dir name is the module name.

Reply:
I will rewrite this piece of code. But it still won't allow you to run the `face_reid` module, because you used a specific volume mapping in the `face_reid` module.